include/crypto/rand.h

```diff
@@ -22,6 +22,16 @@
 # include <openssl/rand.h>
 # include "crypto/rand_pool.h"
 
+# if defined(__APPLE__) && !defined(OPENSSL_NO_APPLE_CRYPTO_RANDOM)
+#  include <Availability.h>
+#  if (defined(__MAC_OS_X_VERSION_MIN_REQUIRED) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 1010) || \
```

Comment:
Should the version number here be 101000?

Availability.h (in the MacOSX10.10 SDK) defines:
```
--snip--
#define __MAC_10_0            1000
#define __MAC_10_1            1010
#define __MAC_10_2            1020
#define __MAC_10_3            1030
#define __MAC_10_4            1040
#define __MAC_10_5            1050
#define __MAC_10_6            1060
#define __MAC_10_7            1070
#define __MAC_10_8            1080
#define __MAC_10_9            1090
#define __MAC_10_10         101000
#define __MAC_10_10_2       101002
#define __MAC_10_10_3       101003
--snip--
```

Reply:
true ... will update both PR.